build/make-distribution.sh

```diff
@@ -42,6 +43,10 @@ while (( "$#" )); do
       NAME="bin-$2"
       shift
       ;;
+    --mvn)
```

Comment:
Would it better to add the usage for this flag to L33, similar to [what Spark does](https://github.com/apache/spark/blob/master/dev/make-distribution.sh#L46)?

Reply:
good call, i added it.